Libraries/Text/Text/RCTTextView.m

```diff
@@ -341,6 +359,26 @@ - (NSString *)accessibilityLabel
 }
 #endif // TODO(macOS GH#774)
 
+# pragma mark - Selection
+
+#if TARGET_OS_OSX // TODO(macOS GH#774)
+- (void)textDidEndEditing:(NSNotification *)notification
+{
+  _textView.selectedRange = NSMakeRange(NSNotFound, 0);
+}
+
+- (void)drawFocusRingMask
```

Comment:
@shwanton Our fork doesn't have drawFocusRingMask (just focusRingMaskBounds), is it needed?

Reply:
see comment on `enableFocusRing` for proposed behavior